Assets/GameAssets/World/WorldScripts/Interactables/Specific Interactables/GeneratorInteractable.cs

```diff
@@ -18,6 +19,28 @@ public class GeneratorInteractable : MonoBehaviour, IPlayerInteractable
     public bool CanGeneratorBeInteracted { get; set; }
     public bool DoesRequireSpanner { get; set; }
 
+    [Tooltip("Moves the sparks and smoke forwards")]
+    private Vector3 aBoost = new Vector3(0, 0, -1);
```

Comment:
Could this name be a more descriptive name? Or could we just use Vector3.back instead of the variable?